apps/transport/lib/transport/gtfs_data.ex

```diff
@@ -79,48 +79,82 @@ defmodule Transport.GTFSData do
     12 => {0.00171661376953125, 0.0011294445140095472}
   }
 
-  def create_gtfs_stops_materialized_view(zoom_level)
-      when is_integer(zoom_level) and zoom_level in 1..12 do
-    {:ok, %{rows: [[count]], num_rows: 1}} =
+  def list_views(pattern) do
+    query = from(v in "pg_matviews")
+
+    query
+    |> select([v], v.matviewname)
+    |> where([v], like(v.matviewname, ^pattern))
+    |> DB.Repo.all()
+  end
+
+  def create_if_not_exist_materialized_views do
+    @zoom_levels
+    |> Enum.each(fn {zoom_level, _} ->
+      create_gtfs_stops_materialized_view(zoom_level)
+    end)
+  end
+
+  def refresh_materialized_views do
+    @zoom_levels
+    |> Enum.each(fn {zoom_level, _} ->
+      refresh_materialized_view(zoom_level)
+    end)
+  end
+
```

Comment:
Le code ci-dessous fait systématiquement "create if not exists" suivi de "refresh" pour que ça reste simple. Il existe de meilleures techniques, mais c'est pas utile vu que ça tourne suffisamment bien pour le moment.